architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -228,32 +262,58 @@ metadata:
   uid: ee2b9405-c68c-11e5-8a99-525400f25e34
 spec: {}
 status:
-  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample
+  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample <2>
   tags:
   - items:
     - created: 2016-01-29T13:40:11Z
       dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
       generation: 1
-      image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
-    tag: latest
+      image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d <3>
+    tag: latest <4>
 ----
+
+<1> The name of the image stream. 
+<2> The image source repository.
+<3> The SHA identifier that this imagestream tag currently references. Resources that reference this imagestream tag use this identifier.
+<4> The image stream tag.
+
 ====
+//Ben: i'm not really sure what purpose status.dockerImageRepository even serves, since each status.tag indicates where the tag comes from. @soltysh ?
+
+
+Using image streams has several significant benefits:
+
+* You can tag, rollback a tag, and quickly deal with images, without having to re-push using the command line.
+* You can trigger Builds and Deployments when new image is pushed to the registry, or schedule the import to do it later.
```

Comment:
Triggering and scheduling should not be mixed in this way. The above sentence suggests you can specify a point in time when an image will be re-imported, but that's not true. Scheduled import is about having the image stream tag checked every X minutes (configured by cluster admin) and if the source image has changed that change will be picked up and reflected in image stream. That in turn can trigger Build and/or Deployment flow.

Reply:
+1

import - refresh what this imagestreamtag points to
trigger - do something when an imagestreamtag changes

import can lead to a trigger, but so can other things.  and import might not cause a trigger if nothing cares about that particular imagestreamtag.